server/utils/crmDisplayHelper.ts

```diff
@@ -1,6 +1,7 @@
 import { CrmResponse } from '@eqApi'
-import { Section, ShowOrHideWhen } from '@crmDisplay'
+import { Section, ShowOrHideWhen, Subsection } from '@crmDisplay'
 import _ from 'lodash'
+import { fieldHasValue } from './utils'
 
 export const includeSection = <T extends CrmResponse>(section: Section, crmResponse: T): boolean => {
```

Comment:
You can remove export, only used in file and can moved to the bottom. The unit test case in crmDisplayHelper.test.ts can be removed.